cmd/gokr-packer/packer.go

```diff
@@ -646,36 +726,36 @@ func logic() error {
 		}
 	}
 
-	baseUrl, err := url.Parse(*update)
-	if err != nil {
-		return err
+	if *update == "yes" {
```

Comment:
Why is this code duplicated here? Is this a merge artifact?

Reply:
Yes, it is a merge artifact m)